03_launch_mgmt_cluster.sh

```diff
@@ -219,6 +224,39 @@ EOF
   popd
 }
 
+#
+# Launch and configure fakeIPA
+#
+function launch_fakeIPA() {
+  # Create a folder to host fakeIPA config and certs
+  mkdir -p /opt/metal3-dev-env/fake-ipa
+  if [[ "${EPHEMERAL_CLUSTER}" == "kind" ]]; then
+    if [[ "${IRONIC_TLS_SETUP}" == "true" ]]; then
```

Comment:
```suggestion
 if [[ "${EPHEMERAL_CLUSTER}" == "kind" && "${IRONIC_TLS_SETUP}" == "true" ]]; then
```

Reply:
Do it only if it fits the 80 char limit.

Reply:
Hmm but the else here is only for the first condition ?  I do not think it will same isn't it ?

Reply:
in the outer if you don't have anything else right now other than the inner "inner" if so it wouldn't change anything in practice.

Reply:
Is the no-op intentional when "${EPHEMERAL_CLUSTER}" == "kind" and "${IRONIC_TLS_SETUP}" == "false"?
If yes then I am fine leaving it like this.